modules/farm_fd2/src/entrypoints/cover_crop_seeding/lib.submitOptional.unit.cy.js

```diff
@@ -1,121 +1,142 @@
 import { lib } from './lib.js';
+import * as farmosUtil from '@libs/farmosUtil/farmosUtil';
 
-describe('Submission using the cover_crop lib.', () => {
-  beforeEach(() => {
-    cy.restoreLocalStorage();
-    cy.restoreSessionStorage();
-  });
+describe('Submission scenarios for cover crop lib', () => {
```

Comment:
A more descriptive title might be something like "Submissions with optional content omitted"